Qapple/Qapple/SourceCode/Feature/0.SignUpFlow/0-4.AuthCodeForm/AuthCodeFormFeature.swift

```diff
@@ -19,18 +19,24 @@ struct AuthCodeFormFeature {
         var isAuthCodeValidate = true
         var isAuthCheckComplete = false
         var isLoading = false
+        @Presents var alert: AlertState<Action.Alert>?
     }
     
     enum Action: BindableAction {
         case backButtonTapped
         case checkAuthCodeButtonTapped
         case reSendMailButtonTapped
         case checkAuthCodeResponse
+        case reSendMailResponse
```

Comment:
개인적으로 `resend`가 한 단어로 보여 아래와 같은 컨벤션이 더욱 익숙해 보입니다!!
한번 suggestion 써보고 싶어서 남겨봅니다 ㅎㅎ.....

```suggestion
        case resendMailResponse
```

Reply:
오 좋은 것 같습니다!! 바로 반영해둘게요 ㅎㅎ